src/main/java/at/hannibal2/skyhanni/features/event/spook/TheGreatSpook.kt

```diff
@@ -53,11 +62,80 @@ object TheGreatSpook {
         if (isTimeLeftEnabled()) config.positionTimeLeft.renderString(displayTimeLeft, posLabel = "Time Left Display")
     }
 
+    private val mathFearMessagePattern by RepoPattern.pattern(
+        "chat.math",
+        "§d§lQUICK MATHS! §r§7Solve: §r§e(?<math>.*)",
```

Comment:
please add a regex test to this regex pattern. search for "regex-test" in the project for templates to copy from